changelogs/fragments/543-support_for_failover_cluster.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - Add support for windows failover cluster disks
```

Comment:
Could you reference the modules that this fixes please, something like `- win_format, win_partition - Add support for Windows failover cluster disks`.